packages/flutter_tools/test/general.shard/license_collector_test.dart

```diff
@@ -246,7 +229,7 @@ void main() {
     licenseCollector = LicenseCollector(fileSystem: fileSystem);
   });
 
-  testWithoutContext('processes dependant licenses according to instructions', () {
+  testWithoutContext('processes dependant licenses according to instructions', () async {
```

Comment:
I know you didn't write this, but "dependent"

Reply:
I didn't write this ... in this PR